No surprise, first attempt on Windows errors with

julia> using Cxx
ERROR: LoadError: LoadError: cglobal: could not find symbol julia_namespace in library libcxxffi
 in anonymous at .\no file:4294967295
while loading C:\j\.julia\v0.4\Cxx\src\cxxstr.jl, in expression starting on line 5
while loading C:\j\.julia\v0.4\Cxx\src\Cxx.jl, in expression starting on line 159

Thanks. That fixes the julia_namespace issue. 
Now I get these warnings

> WARNING: Could not add directory C:/mingw-builds/x64-4.8.1-win32-seh-rev5/mingw64/x86_64-w64-mingw32/include to clang search path!
> WARNING: Could not add directory C:/mingw-builds/x64-4.8.1-win32-seh-rev5/mingw64/lib/gcc/x86_64-w64-mingw32/4.8.1/include/c++ to clang search path!
> WARNING: Could not add directory C:/mingw-builds/x64-4.8.1-win32-seh-rev5/mingw64/lib/gcc/x86_64-w64-mingw32/4.8.1/include/c++/x86_64-w64-mingw32 to clang search path!

It has to fail because I don't have any C:/mingw-builds/...

Don't know if it's related to the warnings but I now get an error on `std` not being known

> julia> cxx"""
>                  void printme(int x) {
>                     std::cout << x << std::endl;
>                  }
>              """
> __cxxjl_1.cpp:2:8: error: use of undeclared identifier 'std'
>       std::cout << x << std::endl;

--- a/src/bootstrap.cpp
+++ b/src/bootstrap.cpp
@@ -370,7 +370,7 @@ DLLEXPORT char *simple_decl_name(clang::NamedDecl *decl)
 }
 
 // For cxxstr
-void *createNamespace(C,char *name)
+DLLEXPORT void *createNamespace(C,char *name)
 {
   clang::IdentifierInfo *Id = Cxx->CI->getPreprocessor().getIdentifierInfo(name);
   return (void*)clang::NamespaceDecl::Create(
@@ -685,7 +685,7 @@ class JuliaCodeGenerator : public clang::ASTConsumer {
 
 
 extern "C" {
-  void *julia_namespace = 0;
+ DLLEXPORT void *julia_namespace = 0;
 }
 
 class JuliaSemaSource : public clang::ExternalSemaSource

